### DISCUSSION TITLE
Transpose(d)?

### DISCUSSION THREAD

**Discussion by SteEcker** [Project-MONAI/MONAI#2142] [General]
Hi, 

Thank you for this great effort to harmonize the medical image processing step! I am new to this project and was wondering if there is a reason why there does not exist a corresponding dictionary method for [Transpose](https://docs.monai.io/en/latest/_modules/monai/transforms/utility/array.html#Transpose)?

For example, when I read my DICOM image series with the LoadImaged method it returns my volumes as (n_slices, h, w) and I want to rearrange the dimensions. 
Please correct me if there exists a method but I missed it.

**Comment by rijobro** [comment]
Hi @SteEcker, I've just submitted a PR for `Transposed` here: https://github.com/Project-MONAI/MONAI/pull/2144.

**Reply by rijobro** [reply]
merged into dev.

**Reply by SteEcker** [reply]
Thanks a lot, that's great!
